src/user/user.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { User } from './entities/user.entity';
+import { Repository } from 'typeorm';
+
+@Injectable()
+export class UserService {
+  constructor(
+    @InjectRepository(User)
+    private userRepository: Repository<User>,
```

Comment:
`readOnly`

Reply:
Not sure if this can be made read only. The NestJs documentation tells you to add it like this. I'll re-check it.

Reply:
Seems to be fine. Changed it.